src/main/java/com/pscheduler/util/parser/CourseFileCreator.java

```diff
@@ -11,6 +11,7 @@
 public class CourseFileCreator {
 
     public static void main(String[] args) throws Exception {
+        System.out.println("CourseFileCreator ran");

```

Comment:
Try not to push these println statements to the branch once you have something working.
Same goes with console.log

Reply:
Done